packages/web/src/dotlottie.ts

```diff
@@ -832,5 +832,22 @@ export class DotLottie {
     this._playbackState = 'stopped';
   }
 
+  /**
+   * Freezes the animation by stopping the animation loop.
+   *
+   */
+  public freeze(): void {
+    this._stopAnimationLoop();
```

Comment:
@theashraf How come _stopAnimationLoop is public but marked with an underscore?

Reply:
@samuelOsborne, nice catch! I've updated all the methods that were supposed to be private.